app/views/layouts/application.html.erb

```diff
@@ -18,6 +19,5 @@
       </footer>
     </div>
   </div>
-  <%= javascript_include_tag "//ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js" %>
```

Comment:
The application.js loads before this line, resulting in an error that requires playing in application.html.erb